javascript/javascript3/week3/gitapi.js

```diff
@@ -0,0 +1,36 @@
+//Global Variables
+const myDiv = document.querySelector(".repos-list");
+
+//functions
+function render(promises) {
+    const userRepos = promises.map(promise => promise.items);
```

Comment:
You can seperate things even more here. It does more than render. Also try to be as specific/declarative as possible. myDiv and myUl should is not descriptive